rules/go_rules.build_defs

```diff
@@ -928,6 +929,14 @@ def go_get(name:str, get:str|list, repo:str='', deps:list=[], exported_deps:list
         else:
             all_installs += [getone]
 
+    # in case of installing packages using other package sources 
+    if srcs == [] and isinstance(get, list) and get == [] and install != [] and binary == False:
```

Comment:
Could we simplify this to: 
```
if not get:
   if not install:
      raise ...
   ...
```
`get` by this point will be converted as such: `get = [get]` unless it's an empty list. I'm not sure what the significance of srcs and binary is? 

I think we'd want to try to do this sort of inference whenever they provide `get = []` i.e. get is falsy?

Reply:
I'm checking for `binary==False` and `srcs==[]`  just to make sure I'm not going to interfere with outs that might be added in the code block that's above in the file.
I will do the other suggested change